sympy/tensor/tensor.py

```diff
@@ -856,7 +856,12 @@ def _get(self, key):
 
         if isinstance(key, TensAdd):
             sumvar = S.Zero
-            data_list = [i.data for i in key.args]
+            data_list = []
+            for i in key.args:
+                if isinstance(i,TensExpr):
```

Comment:
Can you add a space after the comma? SymPy is trying to maintain a standard coding style.